addons/website_blog/static/src/js/tours/website_blog.js

```diff
@@ -88,5 +88,6 @@
         extra_trigger: "iframe body:not(.editor_enable)",
         position: "bottom",
         content: Markup(_t("<b>Publish your blog post</b> to make it visible to your visitors.")),
+        isCheck: true,
```

Comment:
See review point `4.`, don't change this step but add a new one (check:true) for test only (auto:true)